### DISCUSSION TITLE
Is there a way to disable meow-global-mode in specific major modes?

### DISCUSSION THREAD

**Discussion by ghost** [meow-edit/meow#374] [Q&A] [answered]
Hi, I'm stuck on this problem - I need meow mode disabled for vterm mode, I have tried doing it with hook:
`(add-hook 'vterm-mode-hook (lambda () (meow-global-mode 0)))`, but it didn't work. What can you suggest?

**Comment by eshrh** [comment, marked as answer]
Not at the moment (meow-global-mode by nature assumes you have it on for all modes, which has effects on shims and such). You can try something like this though: https://github.com/meow-edit/meow/issues/317#issuecomment-1233622579
